doc/changes/changes_4.0.4.md

```diff
@@ -0,0 +1,5 @@
+# Exasol Virtual Schemas 4.0.4, released 2020-??-??
```

Comment:
Don't forget to change the data before you commit

Reply:
I'm not going to release it yet, so it stays like this until I decide to release